content/recipes/swagger.md

```diff
@@ -315,29 +376,32 @@ async create(@Body() createCatDto: CreateCatDto) {
 }
 ```
 
-To specify a return model for the requests, one has to create a class and annotate all properties with the `@ApiModelProperty()` decorator.
+To specify a return model for the requests, one has to create a class and annotate all properties with the `@ApiProperty()` decorator.
```

Comment:
To specify a return model for the requests, one has to create a class and annotate all properties with the `@ApiProperty()` decorator.

->

To specify a return model for a request, we must create a class and annotate all properties with the `@ApiProperty()` decorator.